modules/sim_2d/sim_2d_component.cc

```diff
@@ -293,28 +272,26 @@ void SimControl::OnPlanning(const std::shared_ptr<ADCTrajectory>& trajectory) {
   }
 }
 
-void SimControl::Freeze() {
+void Sim2DComponent::Freeze() {
   next_point_.set_v(0.0);
   next_point_.set_a(0.0);
   prev_point_ = next_point_;
 }
 
-void SimControl::RunOnce() {
+void Sim2DComponent::SimStep() {
```

Comment:
In the apollo repo, we usually use RunOnce() or Proc(). Would be great if we can follow the convention.